cinema/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()

```

Comment:
I guess you can set blank=True for the description field. But it seems like it is not an argument for reject:)